packages/sit-onyx/playwright.config.ts

```diff
@@ -8,6 +8,11 @@ import { fileURLToPath } from "node:url";
 export default defineConfig({
   testDir: "./src/components",
   snapshotDir: "./playwright/snapshots",
+  // custom snapshotPathTemplate to remove the testFileName folder that we don't want
+  snapshotPathTemplate: "{snapshotDir}/{testFileDir}/{arg}-{projectName}-{platform}{ext}",
+  // we don't want to update snapshots on the local machine of each developer.
+  // if you want to update snapshots for your branch, use the corresponding GitHub action.
```

Comment:
can you pls create a small follow-up PR afterwards with the link to the github action that you create with this PR? ^^ #devsarelazy

Reply:
Yes sure 😄